### PR TITLE
Add eslint-plugin-maintainable config 

### PR DESCRIPTION
**Pull request by asmyshlyaev177**
Just want to share my config, hope it will be useful.

### CODE REVIEW

**Comment by brettz9** [comment]
Although yours is technically a plugin, I think it should be included under configs since it doesn't actually add any of its own rules (that's normally what configs are used for).

**Comment by asmyshlyaev177** [comment]
> Although yours is technically a plugin, I think it should be included under configs since it doesn't actually add any of its own rules (that's normally what configs are used for).

Yeap, other packages do that because can include configs in a plugin, easier than create 2 packages. `Jest` and `Cypress` also come only like plugins and includes configs.

**Comment by brettz9** [comment]
I mean your plugin *only* contains configs, so I think it should be listed as we list other configs--not grouped together with other plugins. The Cypress plugin contains its own unique rules, so it makes sense to be grouped differently.

In general, I think configs are of less interest here than plugins because they don't contain any new behaviors. There are effectively an infinite set of possible configs (which any user can express), whereas plugins with unique rules are more finite, so they deserve more prominent and fine-tuned categorization.

**Comment by asmyshlyaev177** [comment]
Moved it to configs.

**Comment by davidjbradshaw** [comment]
Why is it a plugin and not a config?

**Comment by asmyshlyaev177** [comment]
> Why is it a plugin and not a config?

Because it is convenient, that can include prettier and couple others handy plugins.

**Comment by davidjbradshaw** [comment]
@asmyshlyaev177 that is all possible to do with a config.

**Comment by brettz9** [comment]
A plugin does have the benefit of allowing one to add one's own rules at a later date if one so chooses.

**Comment by asmyshlyaev177** [comment]
> @asmyshlyaev177 that is all possible to do with a config.

Can't, can include config in a plugin but can't put plugin's functionality with config.

**Comment by asmyshlyaev177** [comment]
> @asmyshlyaev177 that is all possible to do with a config.

Official docs, point how to include plugin in config, cause I can't find it.
https://eslint.org/docs/latest/extend/shareable-configs
https://eslint.org/docs/latest/extend/plugins

**Review by brettz9 — CHANGES_REQUESTED** [review]
Seems fine to me if applying the given changes

File: readme.md [hunk 1]
```diff
@@ -70,6 +70,7 @@ If you want to contribute, please read the [contribution guidelines](contributin
 - [Problems](https://github.com/RyanZim/eslint-config-problems) - Shareable config that only catches actual problems, and doesn't enforce stylistic preferences.
 - [Supermind](https://github.com/supermind/eslint-config-supermind) - Shareable config for Supermind style.
 - [Sheriff](https://github.com/AndreaPontrandolfo/sheriff) - Comprehensive and highly opinionated Eslint configuration. Typescript oriented.
+- [eslint-plugin-maintainable](https://github.com/asmyshlyaev177/eslint-plugin-maintainable) - Plugin focused on code quality and simplicity, includes Prettier.
```
[the remark this answers is not in the document]
Comment (asmyshlyaev177) [reply readme.md:73]: Maybe better to close it for now, there are breaking changes with eslint 9, wait till it settle.

**Pull request closed without merging**